### ISSUE TITLE
CommandBar items do not have aria-setsize and aria-posinset attributes

### ISSUE THREAD

**Issue by trevorsg** [OfficeDev/office-ui-fabric-react#3953]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Already done, see #3951 

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:

CommandBar items need to have aria-setsize and aria-posinset so that Narrator will announce their positions within the set.

#### Actual behavior:

They don't have these attributes

#### Expected behavior:

They have these attributes.

**Comment by trevorsg** [comment]
I would think so, although I wouldn't consider myself an expert. I had a bug for it opened by our accessibility vendor team, and they're supposed to be the experts!

**Comment by micahgodbolt** [comment]
And we had an accessibility vendor team log a bug that the props should be removed ...

This is how recursion destroys our sanity :D
